src/Symfony/Component/Console/Command/InvokableCommand.php

```diff
@@ -34,13 +34,15 @@ class InvokableCommand implements SignalableCommandInterface
     private readonly ?SignalableCommandInterface $signalableCommand;
     private readonly \ReflectionFunction $reflection;
     private bool $triggerDeprecations = false;
+    private $callable;
 
     public function __construct(
         private readonly Command $command,
-        callable $code,
+        callable $callable,
```

Comment:
CPP can be used here.
```suggestion
        private callable $callable,
```

Reply:
Surprisingly, PHP does not allow `callable` type hint on a property. See https://wiki.php.net/rfc/typed_properties_v2#supported_types and https://www.php.net/manual/en/language.types.declarations.php#language.types.declarations.base.function. Given this limitation, I think we keep the parameter type hint to callable and not use CPP. Sound OK?